<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUI</class>
 <widget class="QWidget" name="SettingsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RustDaVinci Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>381</width>
     <height>461</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="Line" name="line_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="faqLinkLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>390</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://github.com/alexemanuelol/RustDaVinci/blob/master/docs/FAQ.md&quot;&gt;&lt;span style=&quot; font-family:'Arial Black'; font-weight:600; text-decoration: underline; color:#000000;&quot;&gt;Frequently Asked Questions&lt;/span&gt;&lt;/a&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
    <widget class="QComboBox" name="paintingQualityComboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string>High</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Painting Quality:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Control area width:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in pixels)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in pixels)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="controlAreaYLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearCoordinatesPushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Clear Coordinates</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="controlAreaHeightLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="controlAreaWidthLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Control area x-coordinate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>160</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in pixels)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in pixels)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="rememberControlAreaCoordinatesCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Always remember the paint controls area coordinates</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="controlAreaXLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Control area height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Control area y-coordinate:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoFindControlAreaCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>331</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatically find the paint controls area coordinates (OpenCV)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="paintingTab">
    <attribute name="title">
     <string>Painting</string>
    </attribute>
    <widget class="QLineEdit" name="pauseKeyLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>F10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause Hotkey:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip Color Hotkey:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="skipColorKeyLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>F11</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel Hotkey:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cancelKeyLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Escape</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="skipDefaultColorCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip painting the default background color</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoUpdateCanvasCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatically update the canvas while painting</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoUpdateCanvasWhenCompletedCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatically save the painting when completed</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="useBrushOpacitiesCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Improve paintings by utilizing different brush opacities</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="drawLinesCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw lines if calculated to be faster</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Default background color:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip these colors:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="backgroundColorLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>16</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="skipColorsLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>36, 56, 76</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="experimentalTab">
    <attribute name="title">
     <string>Experimental</string>
    </attribute>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mouse-click delay:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in milliseconds)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mouseClickDelayLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in milliseconds)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Line-draw delay:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineDrawingDelayLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>25</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in milliseconds)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(in pixels)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Control area delay:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum line width:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minimumLineWidthLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="controlAreaDelayLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="paintingBrushTypeComboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Heavy Round</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Light Round</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/icons.qrc">
        <normaloff>:/brushes/light_round.png</normaloff>:/brushes/light_round.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Heavy Round</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/icons.qrc">
        <normaloff>:/brushes/heavy_round.png</normaloff>:/brushes/heavy_round.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium Round</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/icons.qrc">
        <normaloff>:/brushes/medium_round.png</normaloff>:/brushes/medium_round.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Heavy Square</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/icons.qrc">
        <normaloff>:/brushes/heavy_square.png</normaloff>:/brushes/heavy_square.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Painting brush type:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="doubleClickCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Double-click the mouse for improved painting accuracy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(Only recommended if 1920x1080 resolution in-game and 
automatically find the paint control area coordinates is selected)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="useHiddenColorsCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use the hidden color palette</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="aboutTab">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QLabel" name="gitRepoLinkLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/alexemanuelol/RustDaVinci&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; text-decoration: underline; color:#000000;&quot;&gt;The GitHub Repository&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="defaultPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Defaults</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
